UKHomeOffice/cerberus-service
# Cop 9553 - Display target outcomes

## Description
This PR displays the task outcome as a label per task in the completed tab in task list page.
**JIRA Ticket**:https://support.cop.homeoffice.gov.uk/browse/COP-9553

## To Test
- Pull & run against dev
- Navigate to the **COMPLETED** tab
- If a **COMPLETED** task has an `outcome` in the payload it will be rendered as a label under the task title (see screenshot below)

![Task outcome](https://user-images.githubusercontent.com/19333750/149535688-42bb3a76-fadd-487e-b916-c42b7fd30fd7.png)

- If a **COMPLETED** task does not have an `outcome` , no label is rendered (see screenshot below)

![Task outcome - no label](https://user-images.githubusercontent.com/19333750/149536076-216cd461-6db7-41b4-89de-450a82c96a23.png)


## Developer Checklist

\* Required
